bddl/activity_definitions/opening_doors/problem0.bddl

```diff
@@ -7,8 +7,7 @@
         agent.n.01_1 - agent.n.01
     )
     
-    (:init 
-        (closed door.n.01_1)
```

Comment:
We should specify it's not open since the task is opening doors

Reply:
I thought we can't have non-positive predicates in init conditions?

Reply:
(assets are closed by default -- though this is a sim-level implementation detail)